node/prompt-chatgpt/src/main.js

```diff
@@ -16,11 +16,7 @@ export default async ({ req, res }) => {
     return res.json({ ok: false, error: err.message }, 400);
   }
 
-  const openai = new OpenAIApi(
-    new Configuration({
-      apiKey: process.env.OPENAI_API_KEY,
-    })
-  );
+  const openai = new OpenAIApi();
```

Comment:
How do we pass the API Key if this is removed

Reply:
It's auto detected